custom/rx_addendum.txt

```diff
@@ -1,3 +0,0 @@
-Medical Director of XXXX Clinics
```

Comment:
@tmccormi 
Just checking if you group is using these anymore (readme_rx_printtofax.txt and the globals in the custom/readme_search.txt script above)
If not, then we were thinking of yanking this and those globals from the code.